js/dom-on-load.js

```diff
@@ -0,0 +1,126 @@
+/* global $ $$ focusAccessibility getEventKeyName */// dom.js
+/* global debounce */// toolbox.js
+/* global t */// localization.js
+'use strict';
+
+/** DOM housekeeping after a page finished loading */
+
```

Comment:
This file is extracted from dom.js and runs after `load` event. The only changes are: 1) `document` is replaced with `window` for some events so that all listeners are uniform, and 2) listeners on `load` event were "unpacked" as well as one debounced call.